web/models/queries.ex

```diff
@@ -23,6 +24,16 @@ defmodule Entice.Web.Queries do
     end
   end
 
+  def get_account_by_name(name) do
+    case get_account_id(name) do
```

Comment:
same as above, what name is this? also this get_account_id function should be renamed to something signifying what the name is...